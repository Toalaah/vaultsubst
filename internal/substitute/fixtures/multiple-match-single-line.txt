username=@@path=kv/storage/postgres/creds,field=username,b64=true,transform=trim|upper@@,password=@@path=kv/storage/postgres/creds,field=password@@
