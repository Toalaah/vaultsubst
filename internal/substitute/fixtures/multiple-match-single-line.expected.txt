username=POSTGRES,password=4_5tr0ng_4nd_c0mpl1c4t3d_p455w0rd
